Cargo.toml

```diff
@@ -327,32 +327,32 @@ num-bigint-dig = { version = "0.8" }
 auth                          = { version = "0.20", path = "src/auth", package = "google-cloud-auth" }
 gax                           = { version = "0.22", path = "src/gax", package = "google-cloud-gax" }
 gaxi                          = { version = "0.3", path = "src/gax-internal", package = "google-cloud-gax-internal" }
```

Comment:
these are upstream of `wkt`. They need a bump, right?

Reply:
Good call. AFAICT, `gax` exposes `wkt::Any` indirectly, via:

https://docs.rs/google-cloud-gax/latest/google_cloud_gax/error/rpc/enum.StatusDetails.html#variant.Other

That makes the breaking changes in `wkt` breaking in `gax`.

`gax-internal` was already bumped since the last release.

`auth` seems unaffected?  I bumped the patch version for `auth`, to reflect the dependency changes.